lib/browser.ts

```diff
@@ -417,8 +417,8 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * Call waitForAngularEnabled() without passing a value to read the current
    * state without changing it.
    */
-  waitForAngularEnabled(enabled: boolean|wdpromise.Promise<boolean> = null):
-      wdpromise.Promise<boolean> {
+  waitForAngularEnabled(enabled: boolean|wdpromise.Promise<boolean> = null): boolean
```

Comment:
What's the purpose of changing the return type? IMO, changing the return type will make user confusing and they don't know whether the function is async or not. If you want to give a clear/accurate type inference, you should use function overloading.